packages/reducers/src/makeEnhancer.js

```diff
@@ -8,6 +23,29 @@ import composeReducers from './composeReducers';
 
 export const storeInterface = makeStoreInterface('reducers');
 
+const cleanUpReducer = (state, action) => {
+	if (action.type !== '@redux-tools/REDUCERS_EJECTED') return state;
+
+	const namespaceGroup = action.meta.feature || DEFAULT_FEATURE;
+
+	const pathToNamespace = [namespaceGroup, action.meta.namespace];
+	const pathToReducer = when(always(action.meta.namespace), concat(pathToNamespace))([]);
+
+	const cleanReducers = reduce((state, reducerName) =>
+		dissocPath(append(reducerName, pathToReducer), state)
+	)(__, action.payload);
+
+	const cleanEmptyObjects = compose(
+		when(o(isEmpty, path([namespaceGroup])), dissocPath([namespaceGroup])),
+		when(
+			either(o(isEmpty, path(pathToNamespace)), always(action.meta.isFunctionReducer)),
```

Comment:
action.meta.isFunctionReducer - this will fail if `meta` is not in action

Reply:
isFunctionReducer will hopefully be removed anyway (just a note, see my comment in the other file)